admin/src/js/controllers/edit-user.js

```diff
@@ -254,24 +279,63 @@ angular
       return hasPlace && hasContact;
     };
 
+    const validateFacilityHierarchy = () => {
+      const placeIds = $scope.editUserModel.place;
+
+      if (!placeIds || placeIds.length === 1) {
+        return $q.resolve(true);
+      }
+
+      const getPlace = (placeId) => {
+        return DB().get(placeId);
+      };
+
+      return Promise.all(placeIds.map(getPlace))
```

Comment:
I think we should use an _all_docs call here, instead of individually getting every place with a single request.